utilities/PerformanceTesting/GetAverages.py

```diff
@@ -0,0 +1,74 @@
+import sys, argparse, json
+
+def getFilePaths(file_prefix, num_files):
+    files = []
+
+    for i in range(num_files):
+        files.append(f"{file_prefix}{i}.json")
+
+    return files
+
+def getJsons(file_paths):
+    jsons = []
+
+    for file_path in file_paths:
+        f = open(file_path)
+        j = json.load(f)
+        jsons.append(j)
+
+    return jsons
+
+def isFloat(s):
+    try:
+        float(s)
+        return True
+    except ValueError:
+        return False
+
+def findAverageForMetric(jsons, key, idx):
+    num_files = len(jsons)
+    total = 0
+    count = 0
+
+    for j in jsons:
+        if j[key][idx]['name'] == jsons[0][key][idx]['name']:
+            if isFloat(j[key][idx]['value']):
+                total += float(j[key][idx]['value'])
+                count += 1.0
+        else:
+            print("Two of the files have a mismatch in metrics " + jsons[0][key][idx]['name'] + " and " + j[key][idx]['name'])
+            sys.exit()
+
+    if count > 0:
+        return total / float(count)
+
+def calcAverage(file_prefix, num_files, output_file):
+    files = getFilePaths(file_prefix, num_files)
+
+    jsons = getJsons(files)
+    ave_json = jsons[0]
+
+    out = open(output_file, "w")
+
+    for key in jsons[0]:
+        for i in range(len(jsons[0][key])):
+            ave = findAverageForMetric(jsons, key, i)
+            if ave != None:
+                ave_json[key][i]['value'] = str(ave)
+
+    return ave_json
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Arguments for GetAverages')
+    parser.add_argument('--filePrefix', type=str)
+    parser.add_argument('--numFiles', type=int)
+    parser.add_argument('--outputFile', type=str)
```

Comment:
Could we have some help text on these arguments? Also `type=str` is a little redundant because it's the default